catalog/acm/README.md

```diff
@@ -0,0 +1,90 @@
+# GKE ACM blueprint
+
+A GKEHub Membership and ACM enrollment blueprint for Config Sync and Policy Controller for a pre-provisioned GKE cluster
```

Comment:
I think there is some overlap here btw this and the gitops/configsync blueprint. Can we reuse that?

Reply:
There is some overlap but the provisioning works slightly differently. There is overlap in the GSA and KSA being configured but the rest of the resources are different. The GKEHubFeatureMembership resource, for example, is a different way to managing ACM configuration than ConfigManagement.

It would be great to reduce duplication but need to figure out how.